models/action.go

```diff
@@ -500,51 +526,21 @@ func UpdateIssuesCommit(doer *User, repo *Repository, commits []*PushCommit) err
 			}
 		}
 
+		// Change issue status only if the commit has been pushed to the default branch.
+		if repo.DefaultBranch != branchName {
+			return nil
```

Comment:
Shouldn't this be `continue` so that you can do the marking from the rest of the commit line messages?

Reply:
Oops :man_facepalming:, fixed!